test/Concurrency/DeferredSendableChecking/region_based_sendability.swift

```diff
@@ -48,13 +46,12 @@ func test_isolation_crossing_sensitivity(a : A) async {
     foo_noniso(ns0);
 
     //this call consumes ns1
-    await a.foo(ns1);
+    await a.foo(ns1); //expected-warning{{non-Sendable value sent across isolation domains here, but could be accessed later in this function}}
 
     print(ns0);
-    print(ns1); //expected-warning{{Non-Sendable value consumed, then used at this site; could yield race with another thread}}
+    print(ns1); //expected-note{{access here could race with non-Sendable value send above}}
```

Comment:
I wouldn't use the term "above" since it might not be above in terms of source code (think of a loop where this print happened first). 

Personally I think the warning should be on the illegal use and the note should point to the nearby place where the value got sent across isolation domains. That will draw attention to the problem more clearly since notes are not always seen by folks in an IDE like Xcode (they're folded into the warning and you have to expand it)

Reply:
diagnostics will be reworked in a follow up PR, but I do think that the "warning at consumption site" is a good approach because then the warning appears at the place that actually "did concurrency" - this is the point of this PR